config/vault.yml

```diff
@@ -0,0 +1,13 @@
+development: &development
```

Comment:
this seems counter-productive ... we try to configure everything via env vars, so why add another yml file ?